mlir/lib/Dialect/Linalg/Transforms/Vectorization.cpp

```diff
@@ -2285,7 +2285,7 @@ static SmallVector<Value> ofrToIndexValues(RewriterBase &rewriter, Location loc,
   return result;
 }
 
-/// Rewrite a tensor::PadOp into a sequence of EmptyOp, FillOp and
+/// Rewrite a tensor::PadOp into a sequence of EmptyOp, FillOp/GenerateOp and
```

Comment:
as a sequence of 
[ FillOp GenerateOp InsertSlice ]  .

Reply:
Let me revert this change altogether, I will be refactoring this in #111349 and later.